### PR TITLE
[Database metrics] Add SQL command functions

### PR DESCRIPTION
**Pull request by chinandrew**
Summary of changes:
- Add methods for sending relevant SQL queries to the MariaDB. Didn't add tests since they're all one liner commands that go to the docker container.
- Add gitignore

### CODE REVIEW

**Comment by dfarrow0** [comment]
is it customary to include `.gitignore`?

File: src/database_metrics/db-actions.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        Docker Container object where the MariaDB database is running.
+
+    Returns
+    -------
+    2-Tuple of ExecResults from exec_run, which will be the exit code and any output.
+    No output means the command was successful. The first entry of the tuple is the ExecResult of
+    _clear_cache() and the second entry will be the ExecResult from the table clearing query.
+    """
+    clear_tables = container.exec_run(
+        'mysql -uroot -ppass -e '
```
Comment (dfarrow0) [inline src/database_metrics/db-actions.py:62]: noting for posterity that these credentials are defined here: https://github.com/cmu-delphi/operations/blob/2d2ca8b89c7c8d0945369a4cf09698702b1e5f29/dev/docker/database/Dockerfile#L7-L8

**Review by dfarrow0 — COMMENTED** [review]
(just upgrading my previous comment to a review)

**Review by krivard — APPROVED** [review]
lgtm

**Comment by chinandrew** [comment]
> is it customary to include `.gitignore`?

I've seen both, don't feel strongly if we don't want them checked in.

**Pull request merged**